docs/skopeo.1.md

```diff
@@ -91,10 +103,17 @@ Existing signatures, if any, are preserved as well.
 ## skopeo delete
 **skopeo delete** _image-name_
 
-Mark _image-name_ for deletion.  To release the allocated disk space, you need to execute the container registry garabage collector. E.g.,
+Mark _image-name_ for deletion.  To release the allocated disk space, you need login to the container registry server and execute the container registry garbage collector. E.g.,
```

Comment:
'you need login to" to "you need to login into"

Reply:
Changed to `you must login to`

Reply:
That's better yet.